src/test/frontend/deploy/deploy_controller_test.js

```diff
@@ -46,7 +43,7 @@ describe('Deploy controller', () => {
     let result = ctrl.isDeployDisabled();
 
     // then
-    expect(result).toBeTruthy();
+    expect(result).toBe(true);
```

Comment:
I really like this change.  ```toBeTruthy``` was horrible for me :) I will also start using this ```toBe(true)```